### PR TITLE
Fix error message for addTime and deleteTime

### PR DESCRIPTION
**Pull request by PallonCX**
Closes issue #187.

User will receive "No matching free time to be deleted for the chosen person." if:
1. The chosen person has no free time.
2. None of the chosen person's free time matches the specified free times (actually includes the first point).

Screenshot 2024-04-12 at 2 35 42 PM

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/DeleteTimeCommand.java [hunk 1]
```diff
@@ -31,11 +31,14 @@ public class DeleteTimeCommand extends Command {
...
     public static final String COMMAND_WORD = "deleteTime";
 
-    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Deletes a free time to the person identified "
-            + "by the index number used in the displayed person list. "
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Deletes a free time from the person identified "
+            + "by the index number used in the displayed person list. \n"
+            + "Parameters: INDEX (must be a positive integer) "
+            + PREFIX_FREETIMETAG + "FREE TIME TAG...\n"
+            + "Example: " + COMMAND_WORD + " 1 "
             + PREFIX_FREETIMETAG + "Mon:1300-1400";
```
Comment (PallonCX) [inline src/main/java/seedu/address/logic/commands/DeleteTimeCommand.java:39]: Added more detail to the error message, where it is incomplete (gramatically wrong) previously.

File: src/main/java/seedu/address/logic/commands/DeleteTimeCommand.java [hunk 2]
```diff
@@ -98,6 +102,12 @@ private static Person createEditedPerson(Person personToEdit, EditPersonDescript
             }
         }
 
+        // Case 2: None of the specified free time(s) match any of the person's free time
+        if (freeTimeTags.size() == updatedFreeTimeTags.size()) {
+            throw new CommandException(Messages.MESSAGE_NO_MATCHING_FREE_TIME);
+        }
```
Comment (PallonCX) [inline src/main/java/seedu/address/logic/commands/DeleteTimeCommand.java:108]: `freeTimeTags` is the original free time tags, and `updatedFreeTimeTags` is the new free time tags.

If they have the same size after the `deleteTime` command, it means nothing is deleted, and hence implies no matching.

**Comment by PallonCX** [comment]
Fix issue #188.

Similar to `deleteTime` command, if no new free time is stated, the user will receive "No new free time to be added for the chosen person."
Screenshot 2024-04-12 at 3 09 23 PM

Otherwise, it can be added:
Screenshot 2024-04-12 at 3 12 51 PM
Screenshot 2024-04-12 at 3 13 02 PM

**Review by larainezo — APPROVED** [review]
LGTM

**Comment by PallonCX** [comment]
Closes #154.

**Comment by PallonCX** [comment]
Close #148.

**Review by donwong2308 — APPROVED** [review]
LGTM

**Review by JustWeiHao — APPROVED** [review]
LGTM

**Review by nyankoclaws — APPROVED** [review]
LGTM

**Pull request merged**